Azure/azure-cli
# [AppService] Allow Azure Functions Create to accept Python 3.9 parameters

**Description**

Azure Functions team is going to support Python 3.9 in December, we need to allow customers to create Python 3.9 function app in azure-cli before the Python 3.9 announcement.

This is **NOT** an official Azure Functions Python 3.9 release.

cc: @gzuber, @pragnagopa 

**Testing Guide**

Customer should be able to create Python 3.9
1. On Linux Consumption with `az functionapp create -g <resource_group> -n <app_name> -c <consumption_plan_location> -s <storage_account> --os-type linux --runtime python --runtime-version 3.9 --functions-version 3`
2. On Linux Dedicated/Premium with `az functionapp create -g <resource_group> -n <app_name> -p <dedicated_plan_name> -s <storage_account> --os-type linux --runtime python --runtime-version 3.9 --functions-version 3`

**History Notes**


[AppService] az functionapp create: accept "--runtime python" and  "--runtime-version 3.9" as Azure Functions v3 parameter

---

This checklist is used to make sure that common guidelines for a pull request are followed.
